frontend/src/components/login/Login.tsx

```diff
@@ -86,9 +92,15 @@ export function LoginComponent() {
       sessionStorage.setItem("email", res.email.toString());
```

Comment:
paiseh I think we overlooked this yesterday but can you change this to email.toString(); instead of res.email toString()? The response returns just a userid now.

Reply:
This currently affects the sign-up function where I get redirected to the landing page without getting authenticated, and I receive two toasts of successful login and having an error thrown (as res.email is not valid)